jax_galsim/gsobject.py

```diff
@@ -213,15 +207,8 @@ def xValue(self, *args, **kwargs):
         pos = parse_pos_args(args, kwargs, "x", "y")
         return self._xValue(pos)
 
+    @implements(_galsim.GSObject._xValue)
```

Comment:
There are a few formally private but clearly public-facing methods like this one that I went ahead and wrapped.